_posts/2024-01-12-release.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Release 2024.03
+author: Priya
+date: 2024-01-16
+category: Jekyll
+layout: post
+---
+### Optimizations:
+#### Support for Varied Numeric Separators in Column Type Conversion:
+Mammoth now permits diverse numeric separators in column type conversion, accommodating both Indian and US styles. With this update, Mammoth supports conversions in formats such as "x,xx,xxx" and "xx,xx,xxx.
+
+### Defect Fixes:
+* Resolved an issue where the Data Flow Status - Queued section was not visible when branching out to a project.
+* Fixed the problem where renamed date columns were downloaded as text columns instead of date columns.
```

Comment:
Fixed the problem where renamed date columns were represented as text columns instead in the downloaded excel